llama_cpp/server/app.py

```diff
@@ -495,7 +496,7 @@ async def create_chat_completion(
             if body.logit_bias_type == "tokens"
             else body.logit_bias
         )
-
+            
```

Comment:
Does Black or Ruff need to be ran? @tpfau

Reply:
true